x-pack/package.json

```diff
@@ -46,27 +47,44 @@
     "@types/d3-time": "^1.0.7",
     "@types/d3-time-format": "^2.1.0",
     "@types/elasticsearch": "^5.0.30",
+    "@types/file-type": "^5.2.1",
+    "@types/get-port": "^3.2.0 ",
+    "@types/git-url-parse": "^9.0.0",
+    "@types/glob": "^5.0.35",
```

Comment:
`@types/glob` and `glob` are different major versions, and can probably both be upgraded to version 7